src/image-alt-text/opportunityHandler.js

```diff
@@ -62,11 +64,58 @@ export async function syncAltTextSuggestions({ opportunity, newSuggestionDTOs, l
     }
   }
 }
-// TO-DO: Implement in https://jira.corp.adobe.com/browse/ASSETS-47371
-const getProjectedMetrics = () => ({
-  projectedTrafficLost: 3871,
-  projectedTrafficValue: 7355,
-});
+
+const getProjectedMetrics = async ({
+  images, auditUrl, context, log,
+}) => {
+  const CPC = 1; // $1
+  const PENALTY_PER_IMAGE = 0.01; // 1%
+
+  const finalUrl = await getRUMUrl(auditUrl);
+  const INTERVAL = 30; // days
+
+  const rumAPIClient = RUMAPIClient.createFrom(context);
+  const options = {
+    domain: finalUrl,
+    interval: INTERVAL,
+  };
+
+  const results = await rumAPIClient.query('traffic-acquisition', options);
+  log.info(`[${AUDIT_TYPE}]: RUM API results: ${JSON.stringify(results)}`);
+
+  const pageUrlToOrganicTrafficMap = results.reduce((acc, page) => {
+    acc[page.url] = {
+      organicTraffic: page.earned,
+      imagesWithoutAltText: 0,
+    };
+    return acc;
+  });
+  log.info(`[${AUDIT_TYPE}]: Page URL to organic traffic map: ${JSON.stringify(pageUrlToOrganicTrafficMap)}`);
+
+  images.forEach((image) => {
+    const fullPageUrl = new URL(image.pageUrl, auditUrl).toString();
+    if (pageUrlToOrganicTrafficMap[fullPageUrl]) {
+      pageUrlToOrganicTrafficMap[fullPageUrl].imagesWithoutAltText += 1;
```

Comment:
please destructure beforehand:
```suggestion
    const { fullPageUrl: rumFullPageUrl } = pageUrlToOrganicTrafficMap;
    if (!isValidUrl(rumFullPageUrl) {
        rumFullPageUrl.imagesWithoutAltText += 1;
```

Reply:
Not sure the suggestion applies here, we're not trying to check if it's a valid URL. 

We're trying to see if the page where the image lives exists within the RUM traffic results.

Does that make sense?